.github/dependabot.yaml

```diff
@@ -0,0 +1,35 @@
+version: 2
```

Comment:
It looks like the addition of the badge to the README was undone.

Reply:
Indeed. Sorry. Thanks for catching this.